CordovaLib/Classes/Public/CDVURLProtocol.m

```diff
@@ -62,7 +62,7 @@ - (void)startLoading
                 // We have the asset!  Get the data and send it along.
                 ALAssetRepresentation* assetRepresentation = [asset defaultRepresentation];
                 NSString* MIMEType = (__bridge_transfer NSString*)UTTypeCopyPreferredTagWithClass((__bridge CFStringRef)[assetRepresentation UTI], kUTTagClassMIMEType);
-                Byte* buffer = (Byte*)malloc((unsigned long)[assetRepresentation size]);
+                Byte* buffer = (Byte*)calloc((unsigned long)[assetRepresentation size], sizeof(Byte));
```

Comment:
- I cannot see any real benefit to using calloc rather than malloc here, since we know the memory will be overwritten anyway
- While reviewing this change, I noticed a minor opportunity for the following code golf: store the result of `[assetRepresentation size]` which is invoked multiple (2) times in the code here